### DISCUSSION TITLE
SPPS in Large rooms, what is the recommended?

### DISCUSSION THREAD

**Discussion by ardanaz** [Ifsttar/I-Simpa#315] [Q&A]
Hi, im trying to do a simulation with a large room where your volume is more than 7500 m3. What is the recommended for this room type ? Its a large restaurant, a university restaurant at Brazil and this work is for my undergraduated final work. I try to use high number of particles per source but the spps results dont make sense...  The mesh is generated correctly...

**Comment by ardanaz** [comment]
> 
I need increed the simulation length ?

**Comment by ardanaz** [comment]
For this volume geometry , i found good results with 40 million of particles per source and simulation length 0,65 seconds. In my Intel I5 was 12 hours simulation. Best regards..

**Reply by sidway** [reply]
Eita
